tests/integration/test_snapshots.py

```diff
@@ -605,3 +608,97 @@ def test_snapshot_state_on_failure(cluster):
     assert info['raft_snapshot_last_term'] == 0
     assert info['raft_snapshot_size'] == 0
     assert info['raft_snapshot_time_secs'] == -1
+
+
+def test_snapshot_sessions(cluster):
+    cluster.create(3)
+
+    @retry(delay=1, tries=10)
+    def assert_num_sessions(val):
+        assert cluster.node(1).info()["raft_num_sessions"] == val
+        assert cluster.node(2).info()["raft_num_sessions"] == val
+        assert cluster.node(3).info()["raft_num_sessions"] == val
```

Comment:
Btw, we have `wait_for_info_param()` function to retry a few times. Is this `retry` package doing something else? 
Maybe we can also add `wait_for_info_param()` to `Cluster` object. Then, all we have to do is: 

```python
cluster.wait_for_info_param("raft_num_sessions", val);
```

Reply:
if wait wont assert after some timeout, then its probably not good enough.  this is an assert.  I guess the test will eventually timeout, but I'd rather it be a quicker failure

Reply:
What is special about assert?

Just to be clear, `wait_for_info_param()` already exists in `RedisRaft` class: https://github.com/RedisLabs/redisraft/blob/49b292c4f54cb681fda737bed360ea5e5ca35345/tests/integration/sandbox.py#L541

It retries a few times and throws an exception. 

So, we can also add this function to `Cluster` class, something like:

```python
    def wait_for_info_param(self, param, value, exclude=None):
        for _id, node in self.nodes.items():
            if exclude is not None and int(_id) in exclude:
                continue
            node.wait_for_info_param(param, value)
```

Then, we can use it in a single line:

```python
cluster.wait_for_info_param("raft_num_sessions", 2);
```